src/styles/blocks/header.scss

```diff
@@ -0,0 +1,65 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: $basic-background-color;
+  font-family: Roboto, sans-serif;
```

Comment:
you added `font-family` to html so you probably don't need `font-family` here. If you need it, use `inherit`